src/patternfly/components/Select/select.scss

```diff
@@ -236,6 +236,13 @@
   .pf-c-select__toggle-clear + .pf-c-select__toggle-button {
     padding-left: var(--pf-c-select__toggle-button--PaddingLeft);
   }
+
+  .pf-c-select__toggle-arrow {
```

Comment:
Can you use the existing selector on L249? That will also keep the specificity down.

https://github.com/patternfly/patternfly-next/blob/26b59efaefdf9f58f1b7e5611ac503e844d4d33f/src/patternfly/components/Select/select.scss#L249